src/components/MainLayout/MainHeader.tsx

```diff
@@ -369,7 +369,11 @@ const CircleNav = () => {
     ];
 
     if (circle.hasVouching) l.push([paths.vouching(circle.id), 'Vouching']);
-    if (myUser.isCircleAdmin) l.push([paths.adminCircles(circle.id), 'Admin']);
+    if (myUser.isCircleAdmin) {
```

Comment:
I thought about this more and I think we should deviate from the original spec. Instead of putting both of these in the nav bar right now, let's put only the members link, and keep the text as "Admin". The name "Members" isn't quite right yet because it still contains the table of epochs as well. We can still get to the new settings page using the original button (see comment below).